docs/apis-tools/working-with-apis-tools.md

```diff
@@ -87,3 +89,13 @@ type:"link", href:"/optimize/next/apis-tools/optimize-api/optimize-api-authoriza
 :::note
 Additionally, visit our documentation on [Operate](../self-managed/operate-deployment/usage-metrics.md) and [Tasklist](../self-managed/tasklist-deployment/usage-metrics.md) usage metric APIs.
 :::
+
+### Postman
+
+Camunda maintains a set of collections and APIs on Postman to help learn and use the available APIs.
+
+Watch and fork your favorite collections and APIs on the [Camunda Team](https://www.postman.com/camundateam).
+
+:::note
+Collections and APIs are manually updated and not all API functionality may be available. For the most up-to-date API functionality, see the component API reference docs.
```

Comment:
Perhaps we should link to these reference docs here?

Reply:
I thought about that, but the best I could do was link to the section above with all the API cards. Would you happen to have a better suggestion or idea?

Reply:
Our API reference documentation is per component, so we would have many links to reference. Let me add a section header that includes the cards and link to that within the admonition since I agree, something is missing here.